train-core.gemspec

```diff
@@ -37,6 +37,6 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'mixlib-shellout', '>= 2.0', '< 4.0'
   spec.add_dependency 'net-ssh', '>= 2.9', '< 6.0'
   spec.add_dependency 'net-scp', '>= 1.2', '< 3.0'
-  spec.add_dependency 'winrm', '~> 2.0'
-  spec.add_dependency 'winrm-fs', '~> 1.0'
+  spec.add_dependency 'ed25519', '~> 1.2' # ed25519 ssh key support
```

Comment:
If we add these then we need compilation tools in train.

Reply:
Yes, I agree; but those came in from a rebase.  I'll yank them here.